lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -660,6 +661,62 @@ int amf_app_handle_pdu_session_response(
 
   return rc;
 }
+/****************************************************************************
+ **                                                                        **
+ ** Name:    convert_ambr()                                                **
+ **                                                                        **
+ ** Description: Converts the session ambr format from                     **
+ **  one defined in create_pdu_session_response to one defined in          **
+ **  pdu_session_estab_accept message                                      **
+ **                                                                        **
+ ** Inputs:  pdu_ambr_response_unit, pdu_ambr_response_value               **
+ **          ambr_unit, ambr_value                                         **
+ **                                                                        **
+ **      Return:   void                                                    **
+ **                                                                        **
+ ***************************************************************************/
+void convert_ambr(
+    uint32_t* pdu_ambr_response_unit, uint32_t* pdu_ambr_response_value,
+    uint8_t* ambr_unit, uint16_t* ambr_value) {
+  int count = 1;
+  if (*pdu_ambr_response_unit == KBPS) {
+    *pdu_ambr_response_value *= 1000;
+  }
+  uint32_t temp_pdu_ambr_response_value = *pdu_ambr_response_value;
+  if (temp_pdu_ambr_response_value / 1000 == 0) {
+    count = 1;
+  } else {
```

Comment:
You do not need if/else as you can initialize the count based on the `pdu_ambr_response_unit`.

Reply:
Comment addressed.

Reply:
Comment addressed.